ci/check_diff.sh

```diff
@@ -160,11 +169,27 @@ function check_repo() {
     cd $WORKDIR
 }
 
+# Zip up all the diff changes detected by the script
+#
+# Globlas:
```

Comment:
Typo here, though for some reason GitHub isn't giving me ability to suggest line-level changes at the moment